geo/src/lib.rs

```diff
@@ -1,6 +1,18 @@
 #![doc(html_logo_url = "https://raw.githubusercontent.com/georust/meta/master/logo/logo.png")]
 
-//! The `geo` crate provides geospatial primitive types and algorithms.
+//! The `geo` crate provides planar geospatial geometries and algorithms.
```

Comment:
I know we go back and forth on this, but I think it's important to mention planar geometries up front – I call out the availability of non-planar length and distance ops further down.